test/integration/canonicalize/canonical.expected

```diff
@@ -1,3 +1,35 @@
+  $ ../../../../install/default/bin/stanc --print-canonical blocks.stan
+data {
+  int N;
+  vector[N] y;
+}
+parameters {
+  real mu;
+  real<lower=0> sigma;
+}
+model {
+  int j = 0;
+  
+  if (N > 5) {
+    y ~ normal(mu, sigma);
+  }
+  else {
```

Comment:
Any idea why this doesn’t pretty-print such that the else is on the same line? I think we do this for normal if/else

Reply:
Huh. My guess is this is something in the special logic in the pretty printer, but I suspect it should be looked into. I’ll look later today if it’s an easy change or not, but I don’t think you need to worry for this PR